coalib/parsing/FilterHelper.py

```diff
@@ -0,0 +1,38 @@
+from coalib.output.printers.LogPrinter import LogPrinter
+
+from coalib.parsing.filters.LanguageFilter import LanguageFilter
+from coalib.parsing.filters.CanDetectFilter import CanDetectFilter
+from coalib.parsing.filters.CanFixFilter import CanFixFilter
+
+
+class FilterHelper:
+    available_filters = {'language': LanguageFilter,
+                         'can_detect': CanDetectFilter,
+                         'can_fix': CanFixFilter}
+
+    @classmethod
+    def get_all_filters_str(cls, sep=', '):
+        return sep.join(sorted(cls.available_filters))
+
+    @classmethod
+    def is_valid_filter(cls, filter):
+        return filter in cls.available_filters
+
+    @classmethod
+    def get_filtered_bears(cls, filter, args, all_bears=None):
+        if all_bears is None:
+            from coalib.settings.ConfigurationGathering import (
+                get_all_bears)
+            all_bears = get_all_bears(LogPrinter())
+        if not args or len(args) == 0:
+            return all_bears
+        if cls.is_valid_filter(filter):
+            filter_object = cls.available_filters[filter]()
+            return filter_object.filter_bears(all_bears, args)
```

Comment:
I've already removed a unnecessary function from `FilterBase`. Anyways, this is not something filter writers have to worry about. They just gotta implement `filter_section_bears` function for the new filter. This is part of helper class and not base class of filters.